tests/test_sota_checkpoints.py

```diff
@@ -327,6 +340,64 @@ def test_eval(self, sota_checkpoints_dir, sota_data_dir, eval_test_struct: EvalR
         self.color_dict[eval_test_struct.model_name_], is_accuracy_within_thresholds = retval
         assert is_accuracy_within_thresholds
 
+    @pytest.mark.parametrize("eval_test_struct", param_list,
+                             ids=ids_list)
+    def test_openvino_eval(self, openvino, sota_checkpoints_dir, sota_data_dir, ov_data_dir,
+                           eval_test_struct: EvalRunParamsStruct):
+        if openvino is None:
+            pytest.skip()
+        os.chdir(PROJECT_ROOT)
+        onnx_path = os.path.join(PROJECT_ROOT, "onnx")
+        if not os.path.exists(onnx_path):
+            os.mkdir(onnx_path)
+        self.CMD_FORMAT_STRING = "{} examples/{sample_type}/main.py -m {} --config {conf} \
+             --data {dataset}/{data_name} --to-onnx={onnx_path}"
+        self.test = "openvino_eval"
+        onnx_cmd = self.CMD_FORMAT_STRING.format(sys.executable, 'test', conf=eval_test_struct.config_name_,
+                                                 dataset=sota_data_dir,
+                                                 data_name=eval_test_struct.dataset_name_,
+                                                 sample_type=eval_test_struct.sample_type_,
+                                                 onnx_path=os.path.join(onnx_path,
+                                                                        str(eval_test_struct.model_name_ + ".onnx")))
+        if eval_test_struct.resume_file_:
+            resume_file_path = sota_checkpoints_dir + '/' + eval_test_struct.resume_file_
+            onnx_cmd += " --resume {}".format(resume_file_path)
+        else:
+            onnx_cmd += " --pretrained"
+        exit_code, err_str = self.run_cmd(onnx_cmd, cwd=PROJECT_ROOT)
+        if exit_code != 0:
+            self.row_dict[eval_test_struct.model_name_].append(err_str)
+            self.color_dict[eval_test_struct.model_name_] = BG_COLOR_RED_HEX
+            assert False
+
+        onnx_model = self.onnx_model_name(eval_test_struct.model_name_)
+        mo_cmd = "{} mo.py --input_model {} --framework=onnx --data_type=FP16 --reverse_input_channels" \
+                 " --mean_values=[123.675,116.28,103.53] --scale_values=[58.4795,57.1429,57.4713] --output_dir {}"\
```

Comment:
`--mean_values` and `--scale_values` is something that is set up per-model, not globally. For some models, it may even be unnecessary.

Reply:
Are we going to take these values ​​from the sota_checkpoints_eval.json file? Or from another?

Reply:
Yes, let's use the `sota_checkpoints_eval.json` to supply this information for now.